src/screens/Assistant/ResultItem.tsx

```diff
@@ -386,16 +385,44 @@ const tripSummary = (tripPattern: TripPattern, t: TranslateFunction) => {
                 ),
               )
             : ''
-        }`;
+        }
+        
+        ${l.line?.name}
```

Comment:
Ville brukt samme logikk som for visningsnavn her, som er:
`leg.fromEstimatedCall?.destinationDisplay?.frontText ?? leg.line?.name;`

Destination display bruker å være mer riktig navn på rutene.

Reply:
ah, ja! mye bedre!